rpi-eeprom-config

```diff
@@ -327,6 +336,8 @@ images.
     parser.add_argument('eeprom', nargs='?', help='Name of EEPROM file to use as input')
     args = parser.parse_args()
 
+    if not rpi4():
+        exit_error("This tool only works on a Raspberry Pi 4")
```

Comment:
To be more consistent I should maybe exit with no error? or the other way around, and have rpi-eeprom-update exit with an error if !rpi4.

Reply:
I frequently use rpi-eeprom-config under Cygwin to modify the EEPROM config when using rpiboot/usbbot for CM4. A warning in the --edit/--apply modes which wrap rpi-eeprom-update might be useful though

Reply:
Oh! good point. I'll prepare a v2.

Reply:
Thanks, there's a very minimal unit test file to verify the script level commands for patching pieeprom.bin
./test/test-rpi-eeprom-config

rpi-eeprom-update -h needs to work on any host so that help2man can be run on other platforms.